patronload/cli.py

```diff
@@ -24,7 +41,44 @@ def main() -> None:
     )
     logger.info("Running patronload process")
 
-    # Do things here!
+    connection = create_database_connection(config_values)
+    logger.info(
+        "Successfully connected to Oracle Database version : %s", connection.version
+    )
+
+    s3_client = client("s3")
+    delete_objects_from_bucket_with_prefix(
+        s3_client, ".zip", config_values["S3_BUCKET_NAME"], config_values["S3_PATH"]
+    )
+    existing_ids: list[str] = []
+    for patron_type, query_params in {
+        "staff": {"fields": STAFF_FIELDS, "table": "LIBRARY_EMPLOYEE"},
```

Comment:
will this for loop always return staff first? I think this is where the code is determines that staff records should take precedence over student records if both records are present.